src/react/viewport-controller.js

```diff
@@ -7,40 +7,43 @@ const {ViewportControls, TransitionManager} = experimental;
 
 import CURSOR from './utils/cursors';
 
-const propTypes = {
+const TRANSITION_PROP_TYPES = {
```

Comment:
Q: Why is this component being changed? Is it not being deleted?

Reply:
Un-intentional change, and yes, this should be deleted, but before it can be deleted, I also need to update the `layer-browser` example which is using it, will do it in next update to this PR.